GenumKit/Stencil/Contexts.swift

```diff
@@ -195,7 +195,7 @@ extension StringsFileParser {
     }
 
     let strings = entries
-        .sorted() { $0.key < $1.key }
+        .sorted() { $0.key.lowercased() < $1.key.lowercased() }
```

Comment:
Might be better to use `str1.caseInsensitiveCompare(str2)` here. Or anything that would ensure that any string case but also any diacritics and other niceties of Unicode are insensitively-compared properly (even if those are very rare to uncommon in _keys_)